modules/nw-ingress-controller-configuration-proxy-protocol.adoc

```diff
@@ -33,25 +44,44 @@ $ oc -n openshift-ingress-operator edit ingresscontroller/default
 
 . Set the PROXY configuration:
 +
-* If your Ingress Controller uses the hostNetwork endpoint publishing strategy type, set the `spec.endpointPublishingStrategy.hostNetwork.protocol` subfield to `PROXY`:
+* If your Ingress Controller uses the `HostNetwork` endpoint publishing strategy type, set the `spec.endpointPublishingStrategy.hostNetwork.protocol` subfield to `PROXY`:
```

Comment:
Based on the source block below, should the H be capitalized?
```suggestion
* If your Ingress Controller uses the `hostNetwork` endpoint publishing strategy type, set the `spec.endpointPublishingStrategy.hostNetwork.protocol` subfield to `PROXY`:
```

Reply:
:heavy_check_mark:

Reply:
There are the two keywords in the below IC configuration file:
```
apiVersion: operator.openshift.io/v1
kind: IngressController
metadata:
  name: int1
  namespace: openshift-ingress-operator
spec:
  replicas: 1
  endpointPublishingStrategy:
    hostNetwork:
      protocol: PROXY
    type: HostNetwork
  domain: int1.shudi-4v702.qe.devcluster.openshift.com
```

Reply:
Thanks, @ShudiLi